packages/support/resources/views/components/badge.blade.php

```diff
@@ -114,19 +114,29 @@
     @if ($iconPosition === IconPosition::Before)
         @if ($icon)
             <x-filament::icon
-                :alias="$iconAlias"
-                :icon="$icon"
-                :wire:loading.remove.delay="$hasLoadingIndicator"
-                :wire:target="$hasLoadingIndicator ? $loadingIndicatorTarget : null"
-                :class="$iconClasses"
+                :attributes="
+                    \Filament\Support\prepare_inherited_attributes(
+                        new \Illuminate\View\ComponentAttributeBag([
+                            'alias' => $iconAlias,
```

Comment:
@danharrin, these can just be passed as attributes like previously, right?
What's the reason these changed?

Reply:
I just made it consistent with what the rest of the PR did, I think this is slightly more readable when all the attributes are in the same place

Reply:
Fair enough. Now it's inconsistent with the rest of the codebase though.
Personally I prefer the old way, as it was clear which attributes really needed to be passed as `:attributes`, which makes it easier to see when `:attributes` is no longer needed and attributes can be passed separately once one is no longer needed.